src/test/getarg_tests.cpp

```diff
@@ -428,8 +430,9 @@ BOOST_AUTO_TEST_CASE(logargs)
     const auto okaylog_negbool = std::make_pair("-okaylog-negbool", ArgsManager::ALLOW_ANY);
     const auto okaylog = std::make_pair("-okaylog", ArgsManager::ALLOW_ANY);
     const auto dontlog = std::make_pair("-dontlog", ArgsManager::ALLOW_ANY | ArgsManager::SENSITIVE);
+    const auto private_data = strprintf("%04x", GetRand<uint16_t>());
```

Comment:
Why does this need to be random? Given that the test already failed on one platform, how is this not going to introduce a non-deterministic intermittent issue on the same platform (or others)?

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/26473#issuecomment-1308470971).